src/util/DateTime.cpp

```diff
@@ -129,7 +129,7 @@ string DateTime::ToLocalTimeString(const char* formatStr) const
     struct tm localTimeStamp = ConvertTimestampToLocalTimeStruct();
 
     char formattedString[100];
-    std::strftime(formattedString, sizeof(formattedString), formatStr, &localTimeStamp);
```

Comment:
change from `#include <time.h>` to `#include <ctime>`

Reply:
Done